test/sanity/ignore.txt

```diff
@@ -3121,6 +3121,12 @@ lib/ansible/modules/network/cloudengine/ce_link_status.py validate-modules:E326
 lib/ansible/modules/network/cloudengine/ce_link_status.py validate-modules:E337
 lib/ansible/modules/network/cloudengine/ce_link_status.py validate-modules:E338
 lib/ansible/modules/network/cloudengine/ce_link_status.py validate-modules:E340
+lib/ansible/modules/network/cloudengine/ce_lacp.py validate-modules:E322
```

Comment:
Why have these ignore.txt entries been added?

Reply:
Yep,it seem that there is something that has changed, so provider is unnecessary.
This entry should not be added.
Thank you for review.@gundalow